xarray/tests/test_backends.py

```diff
@@ -1045,10 +1048,10 @@ def test_unsorted_index_raises(self):
         with self.roundtrip(ds) as ondisk:
             inds = np.argsort(dim1)
             ds2 = ondisk.isel(dim1=inds)
-            try:
-                print(ds2.randovar.values)  # should raise IndexError in netCDF4
-            except IndexError as err:
-                self.assertIn('first by calling .load', str(err))
+            with pytest.raises(IndexError) as err:
```

Comment:
This modified test is failing in Travis. I think we actually want to keep something similar to what we had before. The idea was that netcdf4 was raising an error that we wanted to be sure we were reraising with a better error message. NetCDF4 stopped raising the error in recent versions,so we can silently ignore this behavior if a newer version of netcdf4 is installed.

Reply:
Makes sense - I'll improve the comment instead 😆